Scripts/Sources/Automation/Eval.swift

```diff
@@ -0,0 +1,329 @@
+import Foundation
+import PathKit
+import xcproj
+
+class Eval {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tevelee/Eval/blob/7d312c8f14aff0bcb5efb221edea46e4e917d2e9/Scripts/Sources/Automation/Eval.swift#L5"></span>Type body should span 200 lines or less excluding comments and whitespace: currently spans 213 lines</td>
    </tr>
  </tbody>
</table>